src/Lykke.Service.ICMAdapter.Core/Throttling/EventsPerSecondPerInstrumentThrottlingManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Lykke.Service.ICMAdapter.Core.Throttling
+{
+    public class EventsPerSecondPerInstrumentThrottlingManager : IThrottling
+    {
+        private readonly Dictionary<string, EventsCounter> _eventsPerInstrument = new Dictionary<string, EventsCounter>();
```

Comment:
Is it used from concurrent code? Shouldn't it be `ConcurrentDictionary`?

Reply:
I tested it multiple times and it seemed to me that orderbooks are handled one by one, i.e. until the current orderbook event is handled, no other requests are handled. That's why I decided to leave it as a plain  Dictionary to avoid to potential performance hit of using ConcurrentDictionary as it seemed we don't need synchronization. If you think we better play it safe and this is actually a multi-threaded code, I will set to ConcurrentDictionary. It is possible my testing was wrong, but I didnt notice multiple threads executing that method (while debugging and examining the Thread Window and also printing some debug messages on screen)